MatchmakingServer/Social/SocialService.cs

```diff
@@ -3,21 +3,23 @@
 using System.Reactive.Subjects;
 
 using H2MLauncher.Core.Party;
+using H2MLauncher.Core.Social;
 
-using MatchmakingServer.Core.Social;
 using MatchmakingServer.Parties;
-using MatchmakingServer.Social;
+using MatchmakingServer.SignalR;
 
 using Microsoft.AspNetCore.SignalR;
 
-namespace MatchmakingServer.SignalR;
+namespace MatchmakingServer.Social;
 
 public class SocialService
 {
     private readonly ConcurrentDictionary<string, Player> _onlinePlayers = [];
     private readonly Subject<IReadOnlyCollection<Player>> _playerStatusChanges = new();
+    private readonly Subject<(Player Player, ConnectedServerInfo? ConnectedServer)> _playerConnectedServerChanges = new();
```

Comment:
record/struct ?

Reply:
I think I might just be biased against tuples in OOP. 
https://dotnetfullstackdev.substack.com/p/class-vs-struct-vs-tuple-vs-struct